.github/workflows/publish-to-pypi-test.yml

```diff
@@ -88,12 +88,12 @@ jobs:
           bump2version patch setup.py
         if: github.ref == 'refs/heads/master'
 
-      - name: Push Commit from Bump2Version
-        uses: ad-m/github-push-action@master
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch: ${{ github.ref }}
-        if: github.ref == 'refs/heads/master'
+      # - name: Push Commit from Bump2Version
```

Comment:
Better to delete lines than just comment out. Should we also delete all the other bump2version stuff if it isn't being used?

Reply:
Yes might as well remove it all, I'll work on moving over to the new solution and remove it then just to avoid any potential side effects from removing it all together